app/assets/javascripts/snowcrash/modules/index_table.js.coffee

```diff
@@ -0,0 +1,157 @@
+    else
+      console.log "The browser doesn't support local storage of settings."
+
+    @selectedColumns ||= ['title', 'created', 'updated']
+
+    that = this
+
+    @$columnMenu.find('a').each ->
+      $link = $(this)
+      if that.selectedColumns.indexOf($link.data('column')) > -1
+        $link.find('input').prop('checked', true)
+
+  resetToolbar: =>
+    $('.js-index-table-actions').css('display', 'none')
+    $('.js-index-select-all #select-all').prop('checked', false)
+
+  onColumnPickerClick: (event) =>
+    $target = $(event.currentTarget)
+    val     = $target.data('column')
+    $input  = $target.find('input')
+
+    if ((idx = @selectedColumns.indexOf(val)) > -1)
+      @selectedColumns.splice(idx, 1)
+      setTimeout ->
+        $input.prop('checked', false)
+      , 0
+    else
+      @selectedColumns.push(val)
+      setTimeout ->
+        $input.prop('checked', true)
+      , 0
+
+    $(event.target).blur()
+    @saveColumnState()
+    @showHideColumns()
+    false
+
+  onDeleteSelected: (element, answer) =>
+    if answer
+      that   = this
+      issueIds = []
+
+      $('.js-items-table').find(@selectedItemsSelector).each ->
+        $row = $(this).parent().parent()
+        $($row.find('td')[2]).replaceWith("<td class=\"loading\">Deleting...</td>")
+        issueIds.push($(this).val())
+
+      $.ajax @$jsTable.data('destroy-url'), {
+        method: 'POST'
+        dataType: 'json'
+        data: {ids: issueIds}
+        success: ->
+          for id in issueIds
+            $("#checkbox_#{that.itemName}_#{id}").closest('tr').remove()
+            $("##{that.itemName}_#{id}").remove()
+
+          if $(that.selectedItemsSelector).length == 0
+            that.resetToolbar()
+
+          # TODO: show placeholder if no items left
+
+        error: ->
+          for id in issueIds
+            $row = $("#checkbox_#{that.itemName}_#{id}").closest('tr')
+            $($row.find('td')[2]).replaceWith("<td class='text-error'>Please try again</td>")
+      }
+
+    # prevent Rails UJS from doing anything else.
+    false
+
+  saveColumnState: ->
+    if Storage?
+      localStorage.setItem(@storageKey(), JSON.stringify(@selectedColumns))
+    else
+      console.log "The browser doesn't support local storage of settings."
+      console.log "Column selection can't be saved."
+
+  showHideColumns: =>
+    that = this
+
+    $.each @$table.find('thead th'), (index, th)->
+      $th = $(th)
+
+      if (column = $(th).data('column'))
+        that.tagColumnIndex ||= index if column == 'tags'
+        if that.selectedColumns.indexOf(column) > -1
+          that.$table.find("td:nth-child(#{index + 1})").css('display', 'table-cell')
+          $th.css('display', 'table-cell')
+        else
+          that.$table.find("td:nth-child(#{index + 1})").css('display', 'none')
+          $th.css('display', 'none')
+
+  storageKey: ->
+    @project ||= $('.brand').data('project')
```

Comment:
just noticed that `$('.brand').data('project')` is undefined in CE
we may use 'ce'  or something similar